lib/ansible/cli/doc.py

```diff
@@ -1281,6 +1281,51 @@ def add_fields(text, fields, limit, opt_indent, return_values=False, base_indent
                 text.append("%s%s:" % (opt_indent, subkey))
                 DocCLI.add_fields(text, subdata, limit, opt_indent + '  ', return_values, opt_indent)
 
+    @staticmethod
+    def _add_seealso(text, seealsos, limit, opt_indent):
```

Comment:
I realize the body of this function is existing code that was relocated, but the function itself is new, so it would be good to have type annotations on the function declaration.